package.json

```diff
@@ -19,11 +19,18 @@
   "devDependencies": {
     "@agiledigital/serverless-sns-sqs-lambda": "^0.1.8",
     "cheerio": "^0.22.0",
-    "jasmine": "^2.5.2",
+    "eslint": "^6.8.0",
+    "eslint-config-prettier": "^6.9.0",
+    "eslint-plugin-prettier": "^3.1.2",
+    "eslint-plugin-sonarjs": "^0.5.0",
+    "jasmine": "^3.5.0",
+    "prettier": "^1.19.1",
     "serverless": "^1.53.0",
     "serverless-offline": "^5.11.0"
   },
   "scripts": {
-    "sls": "serverless"
+    "sls": "serverless",
+    "lint": "eslint src",
+    "build": "sls package"
```

Comment:
Probably better to use the full string `serverless`?

Reply:
Done